json_schemas/framework-agreement-details.json

```diff
@@ -3,6 +3,10 @@
   "additionalProperties": false,
   "minProperties": 1,
   "properties": {
+    "countersignerDetails": {
```

Comment:
What do you think, @allait?

We have some other `{x}Details` keys as well as some other `{x}Name` keys, so both have some kind of precedent. 

I thought we were going under the assumption that we were expecting to have to store more countersigner stuff sometime, so calling this `countersignerDetails` at this point was forward-looking.
If not though, `countersignerName` is probably fine as long as it doesn't clash with anything we're going to be storing with our framework agreements.